libs/src/services/creatorNode/index.js

```diff
@@ -80,6 +80,50 @@ class CreatorNode {
     return null
   }
 
+  /**
+   * Fetches the sync status of a user on a Content Node using the Discovery Node response
+   */
```

Comment:
since this is duplicate logic from the other getSyncStatus() function in this module, would be good to explain why

Reply:
also we should prob just use the `/clock_status()` route - its already exposed in a static function in this class (`getClockValue()`) so i think you could just use that and remove this fn

Reply:
discussed offline - use clock status route as sync status route is legacy + will add comment